integration/containerd/runk/runk-tests.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# Copyright (c) 2021-2022 Kata Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will validate runk with containerd
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+KATACONTAINERS_REPO=${katacontainers_repo:="github.com/kata-containers/kata-containers"}
+RUNK_SRC_PATH="${GOPATH}/src/${KATACONTAINERS_REPO}/src/tools/runk"
+RUNK_BIN_PATH="/usr/local/bin/runk"
+TEST_IMAGE="docker.io/library/busybox:latest"
+CONTAINER_ID="id1"
+
+install_runk() {
+    echo "Install runk"
+    pushd ${RUNK_SRC_PATH}
+    make
+    sudo make install
+    popd
+}
+
+setup() {
+    echo "restart containerd service"
+    sudo systemctl restart containerd
+    echo "pull container image"
+    sudo ctr images pull ${TEST_IMAGE}
+}
+
+test_runk() {
+    echo "start container with runk"
+    sudo ctr run --pid-file ${CONTAINER_ID}.pid --rm -d  --runc-binary ${RUNK_BIN_PATH} ${TEST_IMAGE} ${CONTAINER_ID}
```

Comment:
It's better to configure containerd to use runk like https://github.com/kata-containers/kata-containers/blob/main/src/tools/runk/README.md#configure-containerd-to-use-runk and run
`ctr run --runtime runk ...`

Reply:
It seems that `ctr` and `nerdctl` ignores [plugins."io.containerd.grpc.v1.cri"] configuration because they do not use the CRI API. https://github.com/containerd/containerd/discussions/6162#discussioncomment-2032800